staff/ana-sanjuan/js-arrays/greet-demo-changeble/greet.js

```diff
@@ -0,0 +1,17 @@
+function greet(name, salutation) {
```

Comment:
the greet demo and it's respective tests should not be here

Reply:
Done!